src/accelerate/utils/imports.py

```diff
@@ -210,7 +210,19 @@ def is_tqdm_available():
 
 
 def is_mlflow_available():
-    return _is_package_available("mlflow")
+    package_exists = importlib.util.find_spec("mlflow") is not None
```

Comment:
I haven't tried it out, but could the code not be simplified like this:

```python
def is_mlflow_available():
    # <explanation + reference to issue>
    return _is_package_available("mlflow") or _is_package_available("mlflow-skinny")
```

Reply:
Yes, this is indeed what we should do here.